editor/editor_audio_buses.cpp

```diff
@@ -1405,14 +1368,22 @@ EditorAudioBuses::EditorAudioBuses() {
 void EditorAudioBuses::open_layout(const String &p_path) {
 	EditorNode::get_bottom_panel()->make_item_visible(this);
 
-	Ref<AudioBusLayout> state = ResourceLoader::load(p_path, "", ResourceFormatLoader::CACHE_MODE_IGNORE);
+	const String path = ResourceUID::ensure_path(p_path);
+
+	Ref<AudioBusLayout> state;
+	if (ResourceLoader::exists(path)) {
+		state = ResourceLoader::load(path, "", ResourceFormatLoader::CACHE_MODE_IGNORE);
+	} else {
+		EditorNode::get_singleton()->show_warning(vformat(TTR("There is no '%s' file."), path));
+		return;
+	}
 	if (state.is_null()) {
 		EditorNode::get_singleton()->show_warning(TTR("Invalid file, not an audio bus layout."));
```

Comment:
I know it's not related to your PR, but I would also rename this warning.

```suggestion
		EditorNode::get_singleton()->show_warning(TTR(R"(Can't open audio bus layout: "%s" is not a valid audio bus layout.)"));
```